### PR TITLE
Update loading skeleton

### LINKED ISSUE #2219: PAIRING TICKET: Align eCR Viewer Page and Loading page
## What needs to be done

Right now our eCR Viewer page and loading page are styled separately and the loading skeleton is very out of date. We need to update loading designs to more closely match the viewer and, as much as possible, share styling and components between the two to reduce maintenance overhead as we update viewer components.

## Why it needs to be done

The difference in styling details right now are making the loading experience quite jarring, with components all changing place on the page when loading is complete.

## Acceptance Criteria

- [ ] All styling for large components uses the SAME CSS classes between the loading skeleton and the actual page.
- [ ] Add the header to the skeleton and precisely match the one on the eCR viewer, Please use the same component for both
- [ ] The side nav loading skeleton's dimensions are totally off, so again please use the same CSS classes as the actual page to return to parity
- [ ] Replace all text in the loading side nav with the following list:

eCR Summary
eCR Document
 &nbsp;&nbsp;Patient Info
   &nbsp;&nbsp;&nbsp; (loading box)
 &nbsp;&nbsp; Encounter Info
  &nbsp;&nbsp;&nbsp;  (loading box)
&nbsp;&nbsp;  Clinical Info
&nbsp;&nbsp;&nbsp;    (loading box)
 &nbsp;&nbsp; Lab Info
  &nbsp;&nbsp;&nbsp;  (loading box)
 &nbsp;&nbsp; eCR Metadata
   &nbsp;&nbsp;&nbsp; (loading box)
 &nbsp;&nbsp; Unavailable Info

- [ ] Design review with Sarah to make sure it looks right

## Technical Notes

As much as possible share CSS between loading and real components to reduce the complexity of future updates. The goal is for the transition to be as smooth as possible

## Additional context

This has design review included just so Sarah can see the end result and note if it's working as expected

### PR DESCRIPTION
**Pull request by JNygaard-Skylight**
# PULL REQUEST

## Summary
Updates the formatting and layout of the loading skeleton components to match the current application.

## Related Issue
Fixes #2219 

## Acceptance Criteria
- [ ] All styling for large components uses the SAME CSS classes between the loading skeleton and the actual page.
- [ ] Add the header to the skeleton and precisely match the one on the eCR viewer, Please use the same component for both
- [ ] The side nav loading skeleton's dimensions are totally off, so again please use the same CSS classes as the actual page to return to parity
- [ ] Replace all text in the loading side nav with the following list:

eCR Summary
eCR Document
 &nbsp;&nbsp;Patient Info
   &nbsp;&nbsp;&nbsp; (loading box)
 &nbsp;&nbsp; Encounter Info
  &nbsp;&nbsp;&nbsp;  (loading box)
&nbsp;&nbsp;  Clinical Info
&nbsp;&nbsp;&nbsp;    (loading box)
 &nbsp;&nbsp; Lab Info
  &nbsp;&nbsp;&nbsp;  (loading box)
 &nbsp;&nbsp; eCR Metadata
   &nbsp;&nbsp;&nbsp; (loading box)
 &nbsp;&nbsp; Unavailable Info

- [ ] Design review with Sarah to make sure it looks right

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by JNygaard-Skylight** [comment]
@sarahtress I'm not happy with how the side nav looks while loading, but I wanted you to have a look 
image

**Comment by sarahtress** [comment]
> @sarahtress I'm not happy with how the side nav looks while loading, but I wanted you to have a look image

hmmm yeah... I'm wondering if we don't do anything to the side nav when loading except for remove the h3 sections if having loading boxes is too challenging. it also looks like the h2 hierarchy indent has disappeared and the padding is off, but maybe that's just a WIP?

**Comment by JNygaard-Skylight** [comment]
@sarahtress  How does this look?
Screenshot 2024-09-03 at 10 26 10

**Comment by sarahtress** [comment]
> @sarahtress How does this look? Screenshot 2024-09-03 at 10 26 10

so this looks better, but a lot of the AC seem to be missing here bc the spacing and dimensions of the whole page don't match the current designs (side nav too narrow, gap between side nav and content too large, etc) and then also there should only be one loading box in the side nav, not 2 boxes next to each other (so basically the two boxes next to each other horizontally should be merged)

would you be able to merge the boxes and get all of the sizing/spacing/padding matching the normal page?

also to give a full design review, I'd need to see a video of the loading screen in action bc the problem was largely that the interaction was quite jarring

thanks!

**Comment by JNygaard-Skylight** [comment]
@sarahtress does this look better? 

https://github.com/user-attachments/assets/dc94090e-f653-4343-ab34-6660ed9b1ab1

**Comment by sarahtress** [comment]
> @sarahtress does this look better?
> 
>  Screen.Recording.2024-09-05.at.11.13.47.mov

looking better! if you're able to make the changes below, I think we should be in pretty decent shape:

Screenshot 2024-09-05 at 11 50 30 AM

**Comment by JNygaard-Skylight** [comment]
@sarahtress 

https://github.com/user-attachments/assets/25266701-27b4-4289-97d0-bd2efc8815d3

**Comment by sarahtress** [comment]
> @sarahtress
> 
>  Screen.Recording.2024-09-05.at.11.13.47.mov

hmmm this looks the same as the last version I saw?

**Comment by JNygaard-Skylight** [comment]
> > @sarahtress
> > Screen.Recording.2024-09-05.at.11.13.47.mov
> 
> hmmm this looks the same as the last version I saw?

Oops, here's the correct recording: https://github.com/user-attachments/assets/2fc59b34-55cc-4ead-81fa-bd2cfb41a71e

**Comment by sarahtress** [comment]
> > > @sarahtress
> > > Screen.Recording.2024-09-05.at.11.13.47.mov
> > 
> > 
> > hmmm this looks the same as the last version I saw?
> 
> Oops, here's the correct recording: https://github.com/user-attachments/assets/2fc59b34-55cc-4ead-81fa-bd2cfb41a71e

nice, looks great! the one last thing I'd wanna see is how it looks at a bigger screen size to see if it only is behaving nicely at a small screen size. but I'm also about to sign off for PTO, so if you could just check it for performance at some larger screen sizes that would be great. can consider design review done!

File: containers/ecr-viewer/src/app/view-data/components/LoadingComponent.tsx [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
-import SideNav from "./SideNav";
 import {
   AccordionDiv,
   AccordionH4,
   AccordionSection,
 } from "../component-utils";
+import Header from "@/app/Header";
+import classNames from "classnames";
+import PatientBanner from "./PatientBanner";
+// import SideNav from "./SideNav";
```
Comment (emmastephenson) [inline containers/ecr-viewer/src/app/view-data/components/LoadingComponent.tsx:19]: Can you remove these comments?

File: containers/ecr-viewer/src/app/view-data/components/LoadingComponent.tsx [hunk 2]
```diff
@@ -159,17 +243,39 @@ const AccordionLoadingSkeleton = () => {
  * @returns ECR page loading skeleton
  */
 export const EcrLoadingSkeleton = () => {
+  const _isNonIntegratedViewer =
+    process.env.NEXT_PUBLIC_NON_INTEGRATED_VIEWER === "true";
```
Comment (emmastephenson) [inline containers/ecr-viewer/src/app/view-data/components/LoadingComponent.tsx:247]: Are we usually defaulting to the standalone viewer across the application? If so, I think we should probably do the opposite. Our integrated (NBS) states aren't going to have this env var set in all likelihood, but we shouldn't show any components of the standalone viewer to them.

Anyways this is a bigger question - for now probably sufficient to swap the default to `false`!

**Review by emmastephenson — COMMENTED** [review]
Looking great! I pulled down the changes and the functionality/design is looking good to me. Just a couple of minor comments!

**Review by emmastephenson — APPROVED** [review]
LGTM, thanks for your work on this Josh!

**Pull request merged**